test/JDBC/expected/temp_table.out

```diff
@@ -0,0 +1,9 @@
+-- BABEL-4912 test ALTER TABLE DROP COLUMN
+CREATE TABLE #t1 (a INT, b INT)
+GO
+
+ALTER TABLE #t1 DROP COLUMN b
```

Comment:
can we add more tests, such as

- dropping then recreating the same column multiple times 
- make sure that dependencies aren't broken (so if the column depends on another object, or an object depends on the column that the proper behavior happens)
- more complicated dependencies such as with toast/index/identity etc
- ensure that the table also works (i can select, update, drop from it normally)